ambari-web/app/views/common/modal_popups/log_tail_popup.js

```diff
@@ -40,12 +40,12 @@ App.showLogTailPopup = function(content) {
       logSearchUrl: function() {
         var quickLink = App.QuickLinks.find().findProperty('site', 'logsearch-env'),
             logSearchServerHost = App.HostComponent.find().findProperty('componentName', 'LOGSEARCH_SERVER').get('hostName'),
-            queryParams = '';
+            params = '';
         if (quickLink) {
-          queryParams = 'hosts=' + this.get('content.hostName') + '&components=' + this.get('content.logComponentName')
-            + '&query=%5B%7B"id":0,"name":"path","label":"Path","value":"' + this.get('content.filePath')
-            + '","isExclude":false%7D%5D';
-          return quickLink.get('template').fmt('http', logSearchServerHost, quickLink.get('default_http_port')) + '/#/logs/serviceLogs?' + queryParams;
+          params = 'hosts=' + encodeURIComponent(this.get('content.hostName'))
+            + ';components=' + encodeURIComponent(this.get('content.logComponentName'))
+            + ';query=[{"id":0,"name":"path","label":"Path","value":"' + encodeURIComponent(this.get('content.filePath')) + '","isExclude":false}]';
```

Comment:
Square brackets aren't escaped here, while in `host_progress_popup_body_view.js` and in `logs_view.js` they are